lib/matplotlib/ticker.py

```diff
@@ -154,6 +154,15 @@
 :class:`PercentFormatter`
     Format labels as a percentage
 
+:class:`BinaryIntFormatter`
```

Comment:
That last push was mostly an accident. I am overhauling the code based on @anntzer's advice. Thinking about turning some of it into recipes in the docs.